src/ImageView.jl

```diff
@@ -359,9 +357,9 @@ Compat.@constprop :none function imshow_gui(canvassize::Tuple{Int,Int},
     if !isempty(slicedata)
         players = [player(slicedata.signals[i], axisvalues(slicedata.axs[i])[1]; id=i) for i = 1:length(slicedata)]
         guidict["players"] = players
-        hbox = Box(:h)
+        hbox = GtkBox(:h)
         for p in players
-            push!(hbox, frame(p))
+            #push!(hbox, frame(p))  # FIXME: causes freeze in tests
```

Comment:
Tests are passing now that GtkObservables has been updated, but I'm currently stuck on this code involving `player`. It freezes Julia. Once I figure this out I think this is ready for merging.